flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/JdbcConnectionOptions.java

```diff
@@ -35,16 +35,18 @@
 
 	protected final String url;
 	protected final String driverName;
+	protected final int connectionCheckTimeoutSeconds;
 	@Nullable
 	protected final String username;
 	@Nullable
 	protected final String password;
 
-	protected JdbcConnectionOptions(String url, String driverName, String username, String password) {
+	protected JdbcConnectionOptions(String url, String driverName, String username, String password, int connectionCheckTimeoutSeconds) {
 		this.url = Preconditions.checkNotNull(url, "jdbc url is empty");
 		this.driverName = Preconditions.checkNotNull(driverName, "driver name is empty");
 		this.username = username;
 		this.password = password;
+		this.connectionCheckTimeoutSeconds = connectionCheckTimeoutSeconds;
```

Comment:
Would be better to validate the seconds here again, because this is a public API, users may directly use it in `JdbcSink`.

Reply:
done